spec/hashie/array_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe Array do
+  if RUBY_VERSION >= '2.3.0'
```

Comment:
I don't think you can compare versions like that, use `Gem::Version.new` and then compare? I could be wrong if these are always 3-part strings?